testsuite/utils.py

```diff
@@ -176,3 +178,24 @@ def check_condition(condition, condition_type, status, reason=None, message=None
     ):
         return True
     return False
+
+
+def is_nxdomain(hostname: str):
+    """Returns bool value if hostname has `A` record in DNS."""
```

Comment:
My understanding of this function is that it returns `True` if `hostname` cannot be resolved to an IP address. `False` otherwise. But that comment suggests something very different. Please change that comment (if it is wrong) or please explain to me how come that the comment is correct.

Reply:
You are correct the doc string is wrong. I edited it and added useful information.

Reply:
It makes sense now, thanks!